content/en/news/releases/1.14.x/announcing-1.14/change-notes/index.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Istio 1.14 Change Notes
+linktitle: 1.14.0
+subtitle: Minor Release
+description: Istio 1.14.0 change notes.
+publishdate: 2022-05-11
+release: 1.14.0
+weight: 10
+aliases:
+- /news/announcing-1.14.0
+---
+
+## Traffic Management
+
```

Comment:
Obviously some spacing issues here. I'm not sure if we still try and keep the various items together in terms of added, updated, improved. I know there was discussion of putting higher priority items first. Not sure how priority is determined.

Reply:
There needs to be a bunch of grammar changes as well within the items.

Reply:
I have removed extra empty lines and fixed grammar errors that I saw in the file. The items under a section are currently organized based on added, updated, and improved. I am also sure how to assign priorities to them and sort them based on the priority.